lib/node_modules/@stdlib/blas/ext/base/dcusumkbn/examples/c/example.c

```diff
@@ -27,17 +27,17 @@ int main( void ) {
 	double y[] = { 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0 };
 
 	// Specify the number of elements:
-	const int64_t N = 4;
+	const int N = 4;
 
 	// Specify stride lengths:
-	const int64_t strideX = 2;
-	const int64_t strideY = -2;
+	const int strideX = 2;
+	const int strideY = -2;
 
 	// Compute the cumulative sum:
 	stdlib_strided_dcusumkbn( N, 0.0, x, strideX, y, strideY );
 
 	// Print the result:
-	for ( int64_t i = 0; i < 8; i++ ) {
+	for ( int i = 0; i < 8; i++ ) {
 		printf( "y[ %"PRId64" ] = %lf\n", i, y[ i ] );
```

Comment:
FWIW, as `i` is no longer `int64_t` here, `PRId64` is no longer necessary. You can use `%d`.